packages/protocol/src/defaults/common-defaults.ts

```diff
@@ -131,6 +141,25 @@ export const DEFAULT_DATA: ElementData = {
     ...DEFAULT_SIGNAL_CATCH_DATA,
     ...DEFAULT_MAIL_DATA,
     ...DEFAULT_TRIGGER_DATA,
-    ...DEFAULT_REQUEST_DATA
+    ...DEFAULT_REQUEST_DATA,
+    // Other defaults, not implemented yet, but needed to satisfy TS
+    body: {} as RestBody,
+    query: {} as DbQuery,
+    cache: {} as Cache,
+    javaClass: '',
+    userConfig: '',
+    timeout: {} as JavaTimeout & JavaEventTimeout,
+    guid: '',
+    code: '',
+    throws: {} as ErrorDefinition,
+    link: '',
+    clientId: '',
+    operation: {} as SoapOperation,
+    properties: {},
+    exception: {} as SoapWsProcessException,
+    eventId: '',
+    method: 'GET',
+    response: {} as RestResponse,
+    target: {} as RestTarget
```

Comment:
this is the only downside of this change at the moment...
We need to tell TS about this default data which is currently not used. But with each finished element view we would need to do this anyway, so this is currently only a placeholder until it is replaced by the real defaults when we implement those parts.